tldraw/apps/tldraw-logseq/src/lib/shapes/LogseqPortalShape.tsx

```diff
@@ -434,6 +445,7 @@ export class LogseqPortalShape extends TLBoxShape<LogseqPortalShapeProps> {
         }}
         {...events}
       >
+        <div className=""></div>
```

Comment:
Was this intentional?

Reply:
It is been added to draw some debug place holders on the screen, but I forget to remove it